sklearn/covariance/graph_lasso_.py

```diff
@@ -191,6 +191,7 @@ def graph_lasso(emp_cov, alpha, cov_init=None, mode='cd', tol=1e-4,
     else:
         errors = dict(invalid='raise')
     try:
+        d_gap = np.inf
```

Comment:
I ran into a case where ``max_iter`` is 0. I opened #4134 because I'm not familiar with the details.